### ISSUE TITLE
Player IDs are not removed from Ranking table when removed, leading to Internal Server Error

### ISSUE THREAD

**Issue by marthjod** [marthjod/jeux#50]
```
javax.ejb.EJBTransactionRolledbackException: 
Unable to find de.fhb.jeux.persistence.ShowdownPlayer with id 3
```

```
mysql> select count(distinct playerId) from Ranking;
+--------------------------+
| count(distinct playerId) |
+--------------------------+
|                       32 |
+--------------------------+
1 row in set (0.01 sec)

mysql> select count(*) from Player;
+----------+
| count(*) |
+----------+
|        5 |
+----------+
1 row in set (0.00 sec)
```

**Comment by marthjod** [comment]
Forgotten in 104585496ece2514b0dde678ab6c1b53653a401b.

**Comment by marthjod** [comment]
Fixed in 47e05d6a5dd0dbbef62117c3a0aa303b89a516d8.

**Comment by marthjod** [comment]
`delete from Ranking where playerId not in (select id from Player);`
